frontend/ui/src/app/applications/application-detail.component.html

```diff
@@ -310,6 +310,38 @@ <h3 class="block mb-2 text-lg font-medium text-gray-900 dark:text-white">
                   </div>
                 </div>
 
+                <div class="space-y-4 sm:flex sm:space-x-4 sm:space-y-0 mt-4">
```

Comment:
I think it would be better to move the link to the bottom, as I don't think it's more important than the other fields